airflow/providers/amazon/provider.yaml

```diff
@@ -895,6 +896,28 @@ config:
         type: boolean
         example: "True"
         default: "True"
+  aws_auth_manager:
+    description: |
+      This section only applies if you are using the AwsAuthManager. In other words, if you set
+      ``[core] auth_manager = airflow.providers.amazon.aws.auth_manager.aws_auth_manager.AwsAuthManager`` in
+      Airflow's configuration.
+    options:
+      use_experimental:
+        description: |
+          AWS auth manager is not ready to be used. Turn on this flag to use it anyway.
+          Do that at your own risk since the AWS auth manager is not in an usable state.
+        version_added: 8.11.0
+        type: boolean
+        example: "True"
+        default: "False"
```

Comment:
mmm this feels a bit odd?
The feature itself may be marked as experimental.
I guess my issue is more with the name of the config. I suggest `enable=True/False` rather than `use_experimental=True/False`

Reply:
Sounds good to me